assets/src/edit-story/components/library/textLibrary.js

```diff
@@ -14,52 +14,93 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
 /**
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+
 /**
- * External dependencies
+ * Internal dependencies
  */
-import PropTypes from 'prop-types';
+import { Section, MainButton, Title, SearchInput, Header } from './common';
+import { FontPreview } from './text';
+
+const PRESETS = [
+  {
+    id: 'heading',
+    title: 'Heading',
+    fontSize: 28,
+    fontWeight: 800,
+    fontFamily: 'Ubuntu',
```

Comment:
Let's use `maybeEnqueueFontStyle` from `useFont` to make sure the font stylesheets are loaded properly. Otherwise the presets aren't actually displayed in Ubuntu.